lib/vhpi/VhpiCbHdl.cpp

```diff
@@ -716,6 +721,7 @@ GpiObjHdl *VhpiIterator::next_handle(void)
     }
 
     std::string name = vhpi_get_str(vhpiCaseNameP, obj);
+    std::string fq_name = m_parent->get_fullname() + "." + name;
```

Comment:
Isn't there a situation where `m_parent` will be NULL - for example iterating over the root instance?